lib/v2/grist/featured.js

```diff
@@ -0,0 +1,59 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+module.exports = async (ctx) => {
+    const baseUrl = 'https://grist.org/';
+    const { data: response } = await got(baseUrl);
+    const $ = cheerio.load(response);
+
+    const listItems = $('li.hp-featured__tease')
+        .toArray()
+        .map((item) => {
+            item = $(item);
+            const a = item.find('.small-tease__link');
+            const author = item
+                .find('span.contributor-info__name a.byline-link')
+                .toArray()
+                .map((item) => $(item).text())
+                .join(', ');
+            const image = item.find('img').attr('src');
+            return {
+                title: a.text(),
+                link: String(a.attr('href')),
+                author,
+                itunes_item_image: image,
+            };
+        });
+    const items = await Promise.all(
+        listItems.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const { data: response } = await got(item.link);
+                const $ = cheerio.load(response);
+                const time = $('dt.article-meta__item-label:contains("Published") + dd.article-meta__item-value').text();
+                item.pubDate = timezone(parseDate(time, 'MMMDD,YYYY'), 0);
+                item.description = $('.article__content .article-body')
+                    .clone()
+                    .children('div')
+                    .remove()
+                    .end()
+                    .toArray()
+                    .map((element) => $(element).html())
+                    .join('');
+                item.category = $('dt.article-meta__item-label:contains("Topic") + dd.article-meta__item-value a').attr('href').split('/')[1];
+
+                return item;
```

Comment:
You can extract the slug from `item.link` and use the posts WordPress API `/wp-json/wp/v2/posts` with the `slug` search parameter.